lang/po/Firmware_fr.po

```diff
@@ -830,13 +830,13 @@ msgid ""
 "G-code sliced for a different printer type. Please re-slice the model again."
 " Print cancelled."
 msgstr ""
-"G-code pour un type d'imprimante différent. Découper le modèle a nouveau. "
+"G-code a été préparé pour un niveau différent. Découpez le modèle a nouveau. "
```

Comment:
that's a wrong translation.

"G-code pour un type d'imprimante différent. Découpez le modèle à nouveau."